frontend/src/views/web/permission-denied.html

```diff
@@ -3,15 +3,26 @@
         <div class="col s12 m6 offset-m3">
             <div class="card horizontal ev-md-container ev-card-panel ev-z-depth-5 ">
                 <div class="card-image">
-                    <img src="dist/images/email.png" height="100">
+                    <img src="dist/images/email.png" height="125">
                 </div>
                 <div class="card-stacked">
                     <div class="card-content">
-                        <p><strong>Unable to create or select a team.</strong>
-                            <br> {{perm.emailError}}</p>
+                        <p><strong>Unable to create or select a team.</strong><br>
+                        Hi {{perm.user.username}}!
+                        <br> A <strong>verification</strong> link has been sent to {{perm.user.email}}.
+                        <br><strong>Note:</strong> In order to proceed ahead, verify your account by clicking on the link in the email.
+                        <strong><br> We are always there to help you !</p></strong>
```

Comment:
Make it lighter in color. @Obtin

Reply:
![1](https://user-images.githubusercontent.com/25207205/34643777-667f65b4-f350-11e7-9997-242fc991fa6c.png)

Reply:
Make this text `We are always there to help you !` more lighter and bold

Reply:
![aka2](https://user-images.githubusercontent.com/25207205/34650305-14ab1a2c-f3e5-11e7-8b1f-6f15546f4dff.png)
@aka-jain  is this fine?

Reply:
Yes, better. Looks good now.